src/tests/suite/daos_pool.c

```diff
@@ -536,10 +536,8 @@ pool_properties(void **state)
 	if (arg->myrank == 0) {
 		rc = daos_pool_query(arg->pool.poh, NULL, &info, NULL, NULL);
 		assert_rc_equal(rc, 0);
-		rc = daos_debug_set_params(arg->group, info.pi_leader,
-			DMG_KEY_FAIL_LOC, DAOS_FORCE_PROP_VERIFY, 0, NULL);
-		assert_rc_equal(rc, 0);
 	}
+	test_set_fail_loc(arg, info.pi_leader, DAOS_FORCE_PROP_VERIFY | DAOS_FAIL_ALWAYS);
```

Comment:
Without either `ONCE`, `SOME`, or `ALWAYS`, `daos_fail_loc_set` won't call `d_fault_attr_set`, which causes the fail_loc not to be triggered in my testing.

Reply:
Oh, are you asking about why I chose `ALWAYS` instead of `ONCE`? There's no particular reason; I thought because we are to clear the fail_loc anyway, `ALWAYS` is simpler. I'm fine to change it to `ONCE` if requested.

Reply:
Yes I was wondering about ALWAYS vs. ONCE, because I couldn't spot in the code of that test that ALWAYS would be needed in order to make the test work properly (since there is just a single pool query call affected). I'm OK with either if there is a clear done subsequently. Was just curious from this standpoint.